### ISSUE TITLE
oelint-adv performance is very slow

### ISSUE THREAD

**Issue by jclsn** [Freed-Wu/bitbake-language-server#7]
It sometimes takes seconds for the language-server to update the linting highlights etc. Not sure if this is due to oelint-adv being slow or the server.

**Comment by Freed-Wu** [comment]
I believe https://github.com/priv-kweihmann/oelint-adv/issues/524 can speed up linting.

**Comment by jclsn** [comment]
I think this issue should be reopened. For me the diagnostics are not updating properly. I can delete the faulty lines and the diagnostics still show me an error.

**Comment by Freed-Wu** [comment]
> I can delete the faulty lines and the diagnostics still show me an error.

Try `:w` to refresh?

**Comment by jclsn** [comment]
Yes, even after writing the diagnostic error is shown, although the faulty line is already deleted

![grafik](https://github.com/Freed-Wu/bitbake-language-server/assets/19779511/59c1c395-33c4-4a3e-90f3-87e41de3f546)

**Comment by Freed-Wu** [comment]
OK, please wait upstream's https://github.com/priv-kweihmann/oelint-adv/issues/524#issuecomment-2043546570

**Comment by priv-kweihmann** [comment]
In latest release of 6.5.x the linter supports caching, that should speed up the overall UX heavily.
Please use that
